template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hive/HiveKnoxConfigProvider.java

```diff
@@ -23,9 +23,8 @@
         String realm = kerberosConfigOpt.getRealm();
         String keytab = ConfigUtils.getSafetyValveProperty("hive.server2.authentication.spnego.keytab", "hive.keytab");
         String principal = ConfigUtils.getSafetyValveProperty("hive.server2.authentication.spnego.principal", "HTTP/_HOST@" + realm);
-        return List.of(config(HIVE_SERVICE_CONFIG_SAFETY_VALVE,
-                principal
-                        + keytab));
+        String filePerEvent = ConfigUtils.getSafetyValveProperty("hive.hook.proto.file.per.event", "true");
```

Comment:
Is `filePerEvent` being used anywhere? I see `principal` and `keytab` being used on line 27. I would assume `filePerEvent` needs to be returned in the list somehow?

Reply:
Looks like it is being set differently for the Hive metastore below so not sure if it needs to be set here.

Reply:
I copied the change from old PR incorrectly, will fix, will add filePerEvent